plugin-maven/src/main/java/com/diffplug/spotless/maven/java/PalantirJavaFormat.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016-2022 DiffPlug
+ * Copyright 2016-2023 DiffPlug
```

Comment:
ASF recommends using the interception year so that we don't have to update this stuff again and again.

See https://issues.apache.org/jira/browse/LEGAL-629.

However ... it still depends on the maintainers' willing, so I'm here sharing some information instead of suggesting.